src/airtable/rounds/funding_rounds.js

```diff
@@ -93,13 +93,23 @@ const calculateWinningProposalsForEarmark = (proposals, fundsAvailableUSD, ocean
     let fundsLeft = fundsAvailableUSD
     for (let p of proposals) {
         if( fundsLeft > 0 ) {
-            let usdRequested = p.get('USD Requested')
+            let usdRequested = 0;
+            let basisCurrency = p.get("Basis Currency");
+            let oceanRequested = 0;
+            if(basisCurrency === 'OCEAN') {
+                usdRequested = p.get('OCEAN Requested') * oceanPrice
+                oceanRequested = p.get('OCEAN Requested')
+            } else {
+                usdRequested = p.get('USD Requested')
+                oceanRequested = Math.ceil(usdRequested / oceanPrice)
+            }
+
             let grantCarry = p.get('USD Granted') || 0
             let usdGranted = fundsLeft - ( usdRequested - grantCarry ) > 0 ? usdRequested - grantCarry : fundsLeft
             let oceanGranted = Math.ceil((usdGranted + grantCarry ) / oceanPrice)
             usdGranted = (oceanGranted * oceanPrice)
 
-            p.fields['OCEAN Requested'] = Math.ceil(usdRequested / oceanPrice)
+            p.fields['OCEAN Requested'] = oceanRequested
```

Comment:
shouldn't we also set the `USD Requested` field in case base currency is in `OCEAN`?